package.json

```diff
@@ -30,7 +30,7 @@
   "devDependencies": {
     "@docusaurus/module-type-aliases": "^3.0.1",
     "@docusaurus/tsconfig": "^3.0.1",
-    "prettier": "^3.0.3",
+    "prettier": "3.2.5",
```

Comment:
Why is this being updated?

Reply:
I think I installed it separately because for some reason it wasn't there when I ran `npm install`

Reply:
```suggestion
    "prettier": "^3.2.5",
```

and re-run `npm i`